rllib/env/tests/test_infinite_lookback_buffer.py

```diff
@@ -598,6 +599,53 @@ def test_set_with_complex_space(self):
         check(buffer.data[3], self.buffer_3)
 
 
+@pytest.mark.parametrize(
+    "space",
```

Comment:
This is great @pseudo-rnd-thoughts ! This helps us a lot to test here and at other places in the future all kind of spaces for our fucntionalities.